common/src/test/java/com/netflix/conductor/common/workflow/SubWorkflowParamsTest.java

```diff
@@ -7,9 +7,7 @@
 import javax.validation.Validation;
 import javax.validation.Validator;
 import javax.validation.ValidatorFactory;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
No idea if you'll get this, but you shouldnt actually use wildcards in imports, should be explicit.